server/svix-server/src/db/mod.rs

```diff
@@ -31,3 +35,102 @@ pub async fn run_migrations(cfg: &Configuration) {
     let db = connect(cfg).await;
     MIGRATIONS.run(&db).await.unwrap();
 }
+
+/// Wipe an organization from existence in a way that ensures the operation can be tried again on
+/// failure.
+pub async fn wipe_org(cfg: &Configuration, org_id: OrganizationId) {
+    let db = init_db(cfg).await;
```

Comment:
Use a transaction and commit periodically.

The problem you have is that every seaorm call is making a new db connection (or trying from the pool) and initing it, which is slow.